server/routes/temporaryabsences/checkTemporaryAbsenceController.test.ts

```diff
@@ -13,12 +13,16 @@ let app: Express
 const temporaryAbsencesService = createMockTemporaryAbsencesService()
 const raiseAnalyticsEvent = jest.fn() as RaiseAnalyticsEvent
 
-beforeEach(() => {
+const createAppWithAllRoutes = () => {
```

Comment:
to create the app in each test after setting different config.showBreadCrumb values